parsons/mailchimp/mailchimp.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import re
+from parsons.etl import Table
+from parsons.utilities.api_connector import APIConnector
+
+logger = logging.getLogger(__name__)
+
+
+class Mailchimp():
+    """
+    Instantiate Mailchimp Class
+
+    `Args:`
+        api_key:
+            The Mailchimp-provided application key. Not required if
+            ``MAILCHIMP_API_KEY`` env variable set.
+    `Returns:`
+        Mailchimp Class
+    """
+
+    def __init__(self, api_key):
+        self.api_key = api_key
```

Comment:
Stealing from Eliot's comments on another PR:

When possible, we want Parsons connectors to support configuration via environment variables as well as through the arguments of the `__init__` method. The way this is usually done is by making the arguments optional (ie `api_token=None`) and then using the `parsons.check_env.check` method to try to pull the value from the environment. As an example, see the Rock the Vote connector: https://github.com/move-coop/parsons/blob/14e369ec8aa325bed054fb859fa4a2e8789b2aef/parsons/rockthevote/rtv.py#L48-L50

Reply:
Solved in 70c6309.